spec/jobs/refresh_users_reviewable_counts_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+RSpec.describe Jobs::RefreshUsersReviewableCounts do
+  fab!(:admin) { Fabricate(:admin) }
+  fab!(:moderator) { Fabricate(:moderator) }
+  fab!(:user) { Fabricate(:user) }
+
+  fab!(:group) { Fabricate(:group) }
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:reviewable1) { Fabricate(:reviewable, reviewable_by_group: group, reviewable_by_moderator: true, topic: topic) }
+
+  fab!(:reviewable2) { Fabricate(:reviewable, reviewable_by_moderator: false) }
+  fab!(:reviewable3) { Fabricate(:reviewable, reviewable_by_moderator: true) }
+
+  before do
+    SiteSetting.enable_category_group_moderation = true
+    group.add(user)
+    topic.category.update!(reviewable_by_group: group)
+  end
+
+  describe '#execute' do
+    it "publishes reviewable counts for the specified users" do
+      messages = MessageBus.track_publish do
```

Comment:
`MessageBus.track_publish` accepts an argument where you can specify the channel you're tracking. Right now, MessageBus can be publishing any messages and the assertions will still pass.

Reply:
O this is tricky because you have multiple channels, I would suggest checking the channel names like you did in the other test for higher confidence.

Reply:
done!